doc/changelog/03-notations/11090-master+refactoring-application-printing.rst

```diff
@@ -0,0 +1 @@
+- Fixed #11033: regression in not printing coercions to which is also associated a notation (`#11090 <https://github.com/coq/coq/pull/11090>`_, by Hugo Herbelin).
```

Comment:
```suggestion
- Fixed an 8.10 regression related to the printing of coercions associated to notations
  (`#11090 <https://github.com/coq/coq/pull/11090>`_,
  fixes ` #11033 <https://github.com/coq/coq/issues/11033>`_, by Hugo Herbelin).
```

Reply:
Much better, thanks!

Reply:
@herbelin You had marked this as resolved but had never applied the suggestion.

Reply:
Why I'm afraid of is rather that a force-pushed over your change (See force-pushed over 3a92c42 below). I'm very sorry. I can do fix. I would need to remember to systematically re-pulling before force-pushing.